README.md

```diff
@@ -1,8 +1,29 @@
 # KIE Server OptaPlanner Task Assignment
 An example of assigning tasks on KIE Server using OptaPlanner.
 
-Developing Drools and jBPM
-==========================
+## Requirements
+- JDK 1.8 or above
+- Maven 3.0+
+- node.js 8.12+
+- npm 6.4.1+
+- [WildFly](https://download.jboss.org/wildfly/11.0.0.Final/wildfly-11.0.0.Final.zip)
+- KIE Server war file from [OptaPlanner website](http://www.optaplanner.org/download/download.html) (Execution server -> Any application server, unzip to find the war file)
+
+## Running the demo
+### First start a WildFly instance and deploy KIE artifacts:
+- Download and unzip the WildFly distribution. Let’s call the root of the distribution WILDFLY_HOME. This directory is named after the WildFly version, so for example `wildfly-11.0.1.Final` is the one we're using for this demo.
+- Download kie-server-...-ee7.war and place it into `WILDFLY_HOME/standalone/deployments` as `kie-server.war`.
+- Configure user(s) and role(s). Execute the following command `WILDFLY_HOME/bin/add-user.sh -a -u 'kieserver' -p 'kieserver1!' -ro 'kie-server'`. You can of course choose different username and password, just make sure that the user has role `kie-server`
+- Start the server by running `WILDFLY_HOME/bin/standalone.sh -c standalone-full.xml`. The default URI for KIE Server is `http:/localhost:8080/kie-server/services/rest/server`. Check this location in a web browser to verify the server is running. You will be prompted to type in username and password, use the ones you chose when adding a new user.
```

Comment:
Maybe remove the backticks around KIE server URL? It's more convenient to have a link that I can click than having to copy & paste it.